keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/config/EC2Config.kt

```diff
@@ -47,22 +42,6 @@ import org.springframework.context.annotation.Configuration
 @EnableConfigurationProperties(CanaryConstraintConfigurationProperties::class)
 @ConditionalOnProperty("keel.plugins.ec2.enabled")
 class EC2Config {
-
-  @Bean
```

Comment:
These are no longer needed now that we pull the supported kinds from the `ExtensionRegistry`.